package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@folio/inventory",
-  "version": "13.1.0",
+  "version": "14.0.0",
```

Comment:
@OleksandrHladchenko1 - Version 13 hasn't been published. Version 14 is not needed.